### PR TITLE
Meshoptimizer

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@dmjoy Please review.

File: Dockerfile [hunk 1]
```diff
@@ -65,10 +66,7 @@ RUN apt-get update && \
 ENV CONDA_EXECUTABLE /opt/conda/bin/conda
 RUN curl --silent -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-4.5.4-Linux-x86_64.sh && \
     /bin/bash ~/miniconda.sh -b -p /opt/conda && \
-    rm ~/miniconda.sh && \
-    ${CONDA_EXECUTABLE} update -n base conda && \
-    ${CONDA_EXECUTABLE} clean -tipsy && \
```
Comment (dmjoy) [inline Dockerfile:70]: Strong motivation to remove these two lines?  I can see how leaving off the `update` command may make sense, but may be good to leave in the `clean` command in the interest of reducing final container size (assuming it doesn't cause issues for you).

**Comment by dmjoy** [comment]
LGTM with the one comment regarding the Dockerfile.  Can disregard if there are good reasons to leave those commands out.

**Comment by danlipsa** [comment]
Thanks David! I added the clean back and I am rebuilding / testing the container now.

**Pull request merged**